platform/windows/detect.py

```diff
@@ -210,6 +210,7 @@ def configure_msvc(env, manual_msvc_config):
         env.Append(LINKFLAGS=["/SUBSYSTEM:WINDOWS"])
     else:
         env.Append(LINKFLAGS=["/SUBSYSTEM:CONSOLE"])
+        env.AppendUnique(CPPDEFINES=["WINDOWS_SUBSYSTEM_CONSOLE"])
```

Comment:
This is for MSVC build only, same should be added after line 350 too (for MinGW build).